package.json

```diff
@@ -34,13 +34,21 @@
   },
   "peerDependencies": {
     "@aws-sdk/credential-providers": "^3.201.0",
+    "@mongodb-js/zstd": "^1.1.0",
+    "kerberos": "^2.0.1",
```

Comment:
Unless we plan on leaving this PR open until we begin 6.0 work, we need to include `^1.0.0` as well in the semver for kerberos.

Reply:
Just waiting on team opinion on whether this can go in as is or not. I believe 6.0 is starting now.

Reply:
Updated to `1.0.0 || ^2.0.0`